### DISCUSSION TITLE
css_variables.md mistake?

### DISCUSSION THREAD

**Discussion by juanmifernandez** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#293] [Ideas]
Hi
I think there is an error on this page:
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/css_variables.md
Where it says _--map-card-room*_ I think it should be _--map-card-internal-room*_

Thanks for your wonderful card!

**Comment by PiotrMachowski** [comment]
Everything is ok :) "internal" variables are initialized by "non-internal" ones [here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/src/xiaomi-vacuum-map-card.ts#L965). If you use card-mod you have to specify "non-internal" variables in ha-card, but if you want to apply style to some specific element you have to use "internal" ones

**Comment by juanmifernandez** [comment]
I'm using this to define the style of each room (changing the id)
.room-4-wrapper {
      --map-card-internal-room-outline-line-color: gray;
      --map-card-internal-room-outline-fill-color: rgba(255, 255, 203, 0.2);
      --map-card-internal-room-outline-fill-color-selected: rgba(0, 0, 255, 0.1);
      --map-card-internal-room-icon-background-color: rgba(255,255,0, 0.5)
    }
Are you saying that I could define It globaly for all rooms? 
Thanks again for your work

**Reply by PiotrMachowski** [reply]
Yup, is possible:
```yaml
ha-card {
--map-card-room-outline-line-color: gray;
--map-card-room-outline-fill-color: rgba(255, 255, 203, 0.2);
--map-card-room-outline-fill-color-selected: rgba(0, 0, 255, 0.1);
--map-card-room-icon-background-color: rgba(255,255,0, 0.5)
}
```

**Comment by juanmifernandez** [comment]
He he he, I feel I'm so dumb!!!!
Thanks a lot
